lib/index.js

```diff
@@ -25,12 +25,15 @@ var Client    = require('client'),
 /// });
 /// ```
 ZAFClient.init = function(callback) {
-  var params = Utils.queryParameters(),
+  var queryParams = Utils.queryParameters(),
+      hashParams = Utils.queryParameters(( document.location.hash || '' ).slice(1)),
```

Comment:
possibly, had copied from the source of `Utils.queryParameters` which did the same thing... doesn't hurt does it?